tools/pylint-extensions/azure-pylint-guidelines-checker/dev_requirements.txt

```diff
@@ -1,3 +1,3 @@
-pylint==2.14.5
+pylint==2.16.0
```

Comment:
Does this bump match what we'll run in the azure-sdk-for-python repo once we merge pylint-next?

Reply:
Small edit should be 2.15.8, not bumping to 2.16 in september b/c libraries I think will still be working through their docstrings  -- will update :)